src/Auth0.php

```diff
@@ -664,16 +664,11 @@ public function decodeIdToken(string $idToken, array $verifierOptions = []) : ar
         }
 
         $verifierOptions = $verifierOptions + [
-            // Set a custom leeway if one was passed to the constructor.
             'leeway' => $this->idTokenLeeway,
             'max_age' => $this->transientHandler->getOnce('max_age') ?? $this->maxAge,
+            self::TRANSIENT_NONCE_KEY => $this->transientHandler->getOnce(self::TRANSIENT_NONCE_KEY)
```

Comment:
It makes sense for the storage to know the value is stored under the name of `self::TRANSIENT_NONCE_KEY`. But this section on the left belongs to "verifierOptions". Do you need to use that name again or can you just use "nonce" here, knowing that that is what you are storing?

```suggestion
            'nonce' => $this->transientHandler->getOnce(self::TRANSIENT_NONCE_KEY)
```

Reply:
That constant is just an enum so you don't have to remember the string.

Reply:
That's my point. For `leeway` and `max_age` you don't use any enum and still you're able to pick them up from the other side. It was a suggestion anyway 👍 the code works